### ISSUE TITLE
`wc_admin_unsnooze_admin_notes` events are being needlessly created

### ISSUE THREAD

**Issue by davisshaver** [woothemes/woocommerce#36287]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

`wc_admin_unsnooze_admin_notes` are being created on 10 minute intervals despite no action being attached to this event.

### Expected behavior

`wc_admin_unsnooze_admin_notes` should not be created if they are not needed.

### Actual behavior

`wc_admin_unsnooze_admin_notes` are being created on 10 minute intervals despite no action being attached to this event.

### Steps to reproduce

1. Install WooCommerce
2. Install WP Crontrol
3. Observe there is no action

There is no function `wc_admin_unsnooze_admin_notes` to call so the hook is unnecessary.

### WordPress Environment

Available upon request.

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by adrianduffell** [comment]
I can reproduce this (see screenshot).

Is snooze still an option in the UI? If not, we could just remove the associated snooze code. Otherwise, I think the function name needs to be updated [here](https://github.com/woocommerce/woocommerce/blob/11ebaeede8e83ef4a5550a06c03d4a5f0a5403e8/plugins/woocommerce/src/Admin/Notes/Notes.php#L18-L19).



![Image](https://github.com/user-attachments/assets/4dd3c629-63ef-4ab6-a86f-26c53bb85944)

**Comment by chihsuan** [comment]
> Is snooze still an option in the UI? If not, we could just remove the associated snooze code. Otherwise, I think the function name needs to be updated [here](https://github.com/woocommerce/woocommerce/blob/11ebaeede8e83ef4a5550a06c03d4a5f0a5403e8/plugins/woocommerce/src/Admin/Notes/Notes.php#L18-L19).

Looks like store alerts can still be snoozed in the UI. 

https://github.com/woocommerce/woocommerce/blob/d53c5a222202705dde2a6d017000f9df102c868b/plugins/woocommerce-admin/client/layout/store-alerts/index.js#L205-L229

**Comment by chihsuan** [comment]
Hmm, look like a snoozed note will not be displayed even after the snooze period is over. The `unsnooze_notes()` function is supposed to unsnooze the notes after the snooze period is over. But it is not being called anywhere.

We replaced the scheduler with wp-cron and the action was removed in this PR but we didn't fix the name.

https://github.com/woocommerce/woocommerce-admin/pull/3936/files#diff-9629281bfd9d7bc9363b02940377d6e73f7b122a9bcfa9e1ead3e6a90c196152L52
